docs/sources/configure-client/go-span-profiles.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Go span profiles with trace-to-profiles
+sidebar_label: Go span profiles
+description: Go span profiles with trace-to-profiles
+weight: 100
+---
+
+# Go span profiles with trace-to-profiles
```

Comment:
is this for the Go language? Otherwise, it can be ready as "Go (for) span profiles with traces to profiles" 

Would it be clearer to use something like Span profiles for Go? Also, Traces to profiles isn't hyphenated in our documentation nor in the UI. 

What do you think about this? Please feel free to edit as you see fit

```suggestion
title: Span profiles with Traces to profiles for Go
menuTitle: Span profiles with Traces to profiles (Go)
description: Learn about and configure Span profiles with Trace to profiles in Grafana for the Go language.
weight: 100
---

# Span profiles with Traces to profiles for Go
```

Reply:
yeah this is just for go